test/e2e_node/summary_test.go

```diff
@@ -40,6 +42,16 @@ import (
 var _ = SIGDescribe("Summary API [NodeConformance]", func() {
 	f := framework.NewDefaultFramework("summary-test")
 	ginkgo.Context("when querying /stats/summary", func() {
+		tempSetCurrentKubeletConfig(f, func(initialConfig *kubeletconfig.KubeletConfiguration) {
```

Comment:
I think we should probably create a new e2e test rather than modifying the existing stats tests to run with swap.

We can tag the test with `[Feature:NodeSwap]` or something similar to ensure that this is only run in environments that have swap present.

Reply:
Yes, it would be better, I will add a new swap test. Thanks for the suggestion.